buildSrc/src/main/java/Dependencies.kt

```diff
@@ -27,7 +27,7 @@ object Versions {
     const val disklrucache = "2.0.2"
     const val leakcanary = "1.6.3"
 
-    const val mozilla_appservices = "0.42.2"
+    const val mozilla_appservices = "0.44.0"
```

Comment:
TODO me: run a smoke test against this upgrade to make sure native features are still working fine.

Reply:
Tested with the sample FxA app. Seems fine.